src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ExplicitInitializationCheck.java

```diff
@@ -185,6 +185,9 @@ private static boolean isNumericType(int type) {
     }
 
     /**
+     * Finds if given node contains numeric constant for zero.Returns
```

Comment:
Remove second sentence that just duplicates `@return`.

`Finds` => `Checks`
Find methods usually return position or Object being looked for. This method returns `boolean`.